tests/test_query.py

```diff
@@ -58,14 +58,14 @@ def test_query_should_add_start_date(self):
         q.search_query = "spotify"
         q.start_time = self.datetime_with_timezone(datetime(2012, 12, 28, 9, 1, 22), "UTC")
         query_string = q.build_query_string()
-        self.assertEqual(query_string, "spotify start-date:2012-12-28T09:01:22Z")
+        self.assertEqual(query_string, "spotify start-date:2012-12-28 09:01:22")
```

Comment:
Shouldn't it be `"spotify start-date:\"2012-12-28 09:01:22\""` (surrounded by quotes) now that the timestamp contains a space?

Reply:
I don't think, the request sent will actually include %20 (or +).  Also take a look at what we had before: 4e4ef2bf116deb2b33dd473e658f16b228833e96

Reply:
But yeah, I see our docs includes quoting, but I don't think it's necessary.

Reply:
Hmm, investigated some more. Seems like you are right. We probably have this bug in most of clients. Nice catch.

Reply:
Adding a `T` between the date and the time parts works as well.

Reply:
> Adding a T between the date and the time parts works as well.

Yes, seems to be working. I am thinking we should possibly update our developer site to prefer this over quoting. Should be easier for customers and also easier for us when reading logs.